cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/monitor/sampling/SamplingFetcher.java

```diff
@@ -173,9 +174,20 @@ private void addBrokerMetricSamples(Set<BrokerMetricSample> brokerMetricSamples)
   }
 
   private double estimateCpuUtil(PartitionMetricSample partitionMetricSample) {
-    int cpuId = KafkaMetricDef.resourceToMetricId(Resource.CPU);
-    int networkOutId = KafkaMetricDef.resourceToMetricId(Resource.NW_OUT);
-    return ModelUtils.estimateLeaderCpuUtilUsingLinearRegressionModel(partitionMetricSample.metricValue(cpuId),
-                                                                      partitionMetricSample.metricValue(networkOutId));
+    List<Integer> cpuId = KafkaMetricDef.resourceToMetricIds(Resource.CPU);
+    List<Integer> networkOutId = KafkaMetricDef.resourceToMetricIds(Resource.NW_OUT);
+    Double cpuUsage = sumOfMetrics(partitionMetricSample, cpuId);
+    Double networkOutUsage = sumOfMetrics(partitionMetricSample, networkOutId);
```

Comment:
As a side note, the JavaDoc for `PartitionMetricSample` class seems to be incomplete -- i.e. changes from `v0` to `v1` are not reflected in JavaDoc.